### ISSUE TITLE
Could not detect cloneorg version

### ISSUE THREAD

**Issue by ghost** [tegon/clone-org-repos#1]
bin ❯ npm -v
2.14.4

bin ❯ node -v
v4.1.1

Any ideas why I'm getting this please?

**Comment by tegon** [comment]
I tested here with same versions and it worked correctly
```shell
~  $ npm -v
2.14.4
~  $ node -v
v4.1.1
~  $ cloneorg -v
cloneorg v0.1.1
```
Did you get any errors? What OS are you using? (I'm on OS X 10.10.2)

**Comment by simonsmith** [comment]
Also having this issue:

`node 0.12.7`
`npm 2.11.3`

**Comment by ghost** [comment]
I'm running on OSX:

~ ❯ uname -a
Darwin daemon.local 15.0.0 Darwin Kernel Version 15.0.0: Tue Sep 22 20:33:10 PDT 2015; root:xnu-3247.10.11.1.1~1/RELEASE_X86_64 x86_64
~ ❯

El Capitan Version 10.11.1 Beta (15B22c)

**Comment by ghost** [comment]
Ah - looks like cli version is looking in the wrong place for package.json when run from anywhere else other than install directory?

``` bash
~ ❯ npm install -g clone-org-repos
/usr/local/bin/cloneorg -> /usr/local/lib/node_modules/clone-org-repos/bin/cloneorg
clone-org-repos@0.2.1 /usr/local/lib/node_modules/clone-org-repos
├── merge@1.2.0
├── cli@0.6.6 (exit@0.1.2, glob@3.2.11)
├── request@2.55.0 (forever-agent@0.6.1, aws-sign2@0.5.0, caseless@0.9.0, tunnel-agent@0.4.1, oauth-sign@0.6.0, stringstream@0.0.4, isstream@0.1.2, json-stringify-safe@5.0.1, qs@2.4.2, node-uuid@1.4.3, mime-types@2.0.14, form-data@0.2.0, combined-stream@0.0.7, tough-cookie@2.0.0, http-signature@0.10.1, bl@0.9.4, hawk@2.3.1, har-validator@1.8.0)
└── prompt@0.2.14 (revalidator@0.1.8, pkginfo@0.3.0, read@1.0.7, winston@0.8.3, utile@0.2.1)
~ ❯ pwd
/Users/kdep
~ ❯ cloneorg -h
ERROR: Could not detect cloneorg version
~ ❯ cd /usr/local/lib/node_modules/clone-org-repos                                                                                                                                                                                                                          ⏎
clone-org-repos git:master ❯ cloneorg -h
Usage:
  clone-org-repos [OPTIONS] [ARGS]

Options:
  -p, --perpage [NUMBER] number of repos per page (Default is 100)
  -t, --type [STRING]    can be one of: all, public, private, forks, sources,
                         member  (Default is all)
  -e, --exclude STRING   Exclude passed repos, comma separated
  -o, --only STRING      Only clone passed repos, comma separated
  -r, --regexp BOOLEAN   If true, exclude or only option will be evaluated as a
                         regexp
  -u, --username STRING  Username for basic authentication. Required to
                         access github api
      --token STRING     Token authentication. Required to access github api
  -k, --no-color         Omit color from output
      --debug            Show debug information
  -v, --version          Display the current version
  -h, --help             Display help and usage details
clone-org-repos git:master ❯ pwd
/usr/local/lib/node_modules/clone-org-repos
```

**Comment by tegon** [comment]
Looks like you're right, I'm setting `package.json` with relative path, which is probably causing the issue. I'll take a look and fix soon.
Thanks for reporting that.

**Comment by tegon** [comment]
Just published version `0.2.2` with the fix.
